src/js/angular/graphexplore/controllers/rdf-class-hierarchy.controller.js

```diff
@@ -57,16 +57,16 @@ function RdfClassHierarchyCtlr($scope, $rootScope, $location, $repositories, $li
     let selectedGraph = allGraphs;
 
     const initView = function () {
-        if (!$scope.getActiveRepository()) {
+        if (!$scope.getActiveRepository() && !$licenseService.isLicenseValid()) {
```

Comment:
Are you sure that the conjunction should be AND and not OR here?

Reply:
Should be OR. Will fix.